src/server/scripts/Northrend/zone_grizzly_hills.cpp

```diff
@@ -599,19 +604,19 @@ class npc_venture_co_straggler : public CreatureScript
                     case EVENT_STRAGGLER_4:
                         me->DisappearAndDie();
                         break;
-                    case EVENT_CHOP:
-                        if (UpdateVictim())
-                            DoCastVictim(SPELL_CHOP);
-                        _events.ScheduleEvent(EVENT_CHOP, 10000, 12000);
-                        break;
+		    case EVENT_CHOP:
```

Comment:
tabs2spaces

Reply:
I don't understand, if i tab it isn't in line with the rest.

Reply:
Forgive my ignorance, i was doing changes directly through the website and not with gitextensions :$